src/pages/Aboutpage/Aboutpage.css

```diff
@@ -0,0 +1,7 @@
+body {
```

Comment:
why are you changing the body? you shouldn't do that all!
if you want to change the background color you can add a background color to the container div in the about page like `bg-black`